src/ConfigManager.tsx

```diff
@@ -20,10 +20,26 @@ import React, { createContext, useContext, useMemo } from 'react';
 
 import type { DashboardConfig } from './types';
 
+type LoginType = 'oauth' | 'token';
```

Comment:
This could be just `keyof AuthOptions`

Reply:
actually this generates a circular dependency in types.
LoginType -> AuthType -> DefaultAuth -> LoginType